ref/android-features.md

```diff
@@ -0,0 +1,21 @@
+Anbox Cloud implements support for various features Android has. The following provides an overview which features are currently avilable as part of Anbox Cloud. If a feature is not listed, it is currently not supported.
+
+| Feature           | Supported | Notes  |
+|--------------------|:-----------------:|
+| [OpenGL ES](https://source.android.com/devices/graphics/arch-egl-opengl) <= 3.2   |  ✓  | |
+| [Vulkan](https://source.android.com/devices/graphics/arch-vulkan) |  ✓  | |
+| [Camera](https://source.android.com/devices/camera) |  ✓  | |
```

Comment:
Picture capture and video shooting are not available for Anbox Cloud deployment where AMD or Intel GPU is installed.

Reply:
Really? That should work.

Reply:
No, they don't work. As far as I tested in the camera application only the camera preview works. Can you double check that? or could you point me to the MP that you got this fixed recently? I don't recall exactly.

Reply:
I've tested that on latest images and you're right camera support is still broken for AMD and Intel GPUs. I had this working but looks like we didn't merge the patches.